env.sh

```diff
@@ -0,0 +1,2 @@
+export REACT_APP_BACKEND_URL=
+export PG_CONN_STRING=
```

Comment:
This file should be copied to the env location, then replaced/provisioned with env variables from Ansible.